### ISSUE TITLE
Final newline does not produce prompt

### ISSUE THREAD

**Issue by rob-smallshire** [sixty-north/gjenta#38]
This Python code in a file `references.py`, which includes a terminating newline:

```
x = 1000
x = 500
y = x
x = 3000

```

when run with `gjenta quickplay -s python references.py`

produces this output:

```
>>> x = 1000
>>> x = 500
>>> y = x
>>> x = 3000

```

but I would expect:

```
>>> x = 1000
>>> x = 500
>>> y = x
>>> x = 3000
>>> 
```

including the last Python prompt as the final line was "entered". I can get the desired effect by including two blank lines at the end of the file, thought this is not what I would expect to need.

I think this is important because it doesn't look to me like the line has executed until I see that final Python prompt.

**Comment by rob-smallshire** [comment]
I just discovered ``--keep-trailing-prompt`.

I'd like to advocate for that being the default.

**Comment by abingham** [comment]
> I'd like to advocate for that being the default.

Yes, this definitely should be the default.

**Comment by abingham** [comment]
Fixed in fcad324387e2eecd59054aa87a3f9e13ee4db14c
